.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+# Checklist:
+
+- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
+- [ ] Any dependent changes have already been merged
```

Comment:
What is this about? Should we keep this?

Reply:
not sure, what do you think? I added this considering the current experience Kara's PR was on hold to merge before your PR of versioning. In these cases, I think good to have in description.

Reply:
OTOH, shall we also mention to sign the commit?

Reply:
> not sure, what do you think? I added this considering the current experience Kara's PR was on hold to merge before your PR of versioning. In these cases, I think good to have in description.

Ah that makes sense :+1: 

> OTOH, shall we also mention to sign the commit?

Yes! Link to the doc which talks abt it.